about.html

```diff
@@ -0,0 +1,120 @@
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">News</a></li><hr>
+                    </ul>
+              </div>
+                <div class="desk-links">
+                    <ul class="deskul">
+                        <li class="desklist"><a href="#" class="desklink"><i class="fa-brands fa-facebook-f"></i></a></li>
+                        <li class="desklist"><a href="#" class="desklink"><i class="fa-brands fa-twitter"></i></a></li>
+                        <li class="desklist"><a href="#" class="desklink">English</a></li>
+                        <li class="desklist"><a href="#" class="desklink">My page</a></li>
+                        <li class="desklist"><a href="#" class="desklink">Logout</a></li>
+                    </ul>
+                </div>
+                <div class="desk-nav">
+                    <div class="logolink">
+                      <a class="logolink1" href="#"><span class="desklogo">VR Odyssey</span></a>
+                    </div>
+                    <ul class="deskul2">
+                      <li class="desklist2"><a href="#" class="desklink1">About</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">program</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Join</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Sponser</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">News</a></li>
+                    </ul>
+                    <div class=desbutton><button class="deskbutton">VR Campaign</button></div>
+                </div>
+                <div class="home">
+                    <ul class="homeul">
+                      <li class="homeli"><a href="./index.html" class="desklink2"><i class="fa-solid fa-house homeicon"></a></i></li>
+                      <li class="homeli"><a href="#" class="desklink2"><i class="fa-solid fa-angle-right"></a></i></li>
+                      <li class="homeli"><a href="./about.html" class="desklink2">about</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <section class="page">
+            <p class="para">"Welcome to VR!"</p>
+            <div class="clip">
+                <h1 class="heading1">VR EVOLUTION:<br>THE NEXT CHAPTER 2023</h1>
+            </div>
+            <div class="paraborder">
+                <p class="para1">We are excited to bring together professionals from across the VR industry for this two-day event exploring the latest trends, innovations, and challenges in VR technology. From keynote speeches and panel discussions to workshops, demonstrations, and networking events, we have designed an engaging and informative program that will provide attendees with valuable insights and practical skills.</p>
```

Comment:
- [ ] Kindlt 😃 try here to increase the font size of this text to be like the Behance I see that it is less than it, so please try to increase it to be like the Behance 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225891294-30d4ce92-3be7-4147-a4b0-c36a753199fe.png)   |    ![image](https://user-images.githubusercontent.com/83360636/225891240-a238b5e6-2038-4522-9f4a-dcb27788540f.png) |